llvm-spirv/lib/SPIRV/libSPIRV/SPIRVValue.cpp

```diff
@@ -75,45 +75,10 @@ bool SPIRVValue::hasNoSignedWrap() const {
   return hasDecorate(DecorationNoSignedWrap);
 }
 
-void SPIRVValue::setNoSignedWrap(bool HasNoSignedWrap) {
-  if (!HasNoSignedWrap) {
-    eraseDecorate(DecorationNoSignedWrap);
-  }
-  if (Module->isAllowedToUseExtension(
-          ExtensionID::SPV_KHR_no_integer_wrap_decoration)) {
-    // NoSignedWrap decoration is available only if it is allowed to use SPIR-V
-    // 1.4 or if SPV_KHR_no_integer_wrap_decoration extension is allowed
-    // FIXME: update this 'if' to include check for SPIR-V 1.4 once translator
-    // support this version
-    addDecorate(new SPIRVDecorate(DecorationNoSignedWrap, this));
-    SPIRVDBG(spvdbgs() << "Set nsw for obj " << Id << "\n")
-  } else {
-    SPIRVDBG(spvdbgs() << "Skip setting nsw for obj " << Id << "\n")
-  }
-}
-
 bool SPIRVValue::hasNoUnsignedWrap() const {
   return hasDecorate(DecorationNoUnsignedWrap);
 }
 
-void SPIRVValue::setNoUnsignedWrap(bool HasNoUnsignedWrap) {
```

Comment:
I suppose these need to be removed as we move to 1.4?

Reply:
Not sure if I understand the comment. The code is removed, isn't it?